core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts

```diff
@@ -199,6 +199,21 @@ export class ExplorationFooterComponent {
     return this.conceptCardManagerService.isConceptCardViewable();
   }
 
+  getCheckpointsCardIndexs(): number[] {
```

Comment:
I don't think this function belongs here, since it's used in the lesson-information-card-modal. Let's move it there

Reply:
Function was changes to lesson-information-card-modal.component.ts